src/domain/wallet-invoices/wallet-invoice-receiver.ts

```diff
@@ -0,0 +1,50 @@
+import { WalletCurrency, ZERO_CENTS, ZERO_SATS } from "@domain/shared"
+
+export const WalletInvoiceReceiver = async ({
+  walletInvoice,
+  receivedBtc,
+  usdFromBtc,
+  usdFromBtcMidPrice,
+}: WalletInvoiceReceiverArgs): Promise<
+  WalletInvoiceReceiver | DealerPriceServiceError | ValidationError
+> => {
+  const zeroBankFee = {
```

Comment:
this should come from config

Reply:
Changed: https://github.com/GaloyMoney/galoy/pull/1241/commits/244ebeaaab6f0677d1a9fa9758536217049ccd6e